CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 * Deprecate `Deprecation.calcInterp` since it was never actually emitted as a
   deprecation.
 
+### Embedded Sass
+
+* Fix a rare race condition in the embedded compiler that could be triggered by
+  a protocol error and cause a crash.
```

Comment:
Technically the issue is not a crash, but a VM freeze that prevent compiler to exit unless getting killed:

```
Unhandled exception:
Bad state: Shouldn't receive a message before being checked out.
#0      _defaultOnData (package:sass/src/embedded/reusable_isolate.dart:141)
#1      _RootZone.runUnaryGuarded (dart:async/zone.dart:1594)
#2      _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:339)
#3      _BufferingStreamSubscription._add (dart:async/stream_impl.dart:271)
#4      _SyncStreamControllerDispatch._sendData (dart:async/stream_controller.dart:784)
#5      _StreamController._add (dart:async/stream_controller.dart:658)
#6      _StreamController.add (dart:async/stream_controller.dart:606)
#7      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:189)


Attempt:11 waiting for isolate vm-isolate to check in
Attempt:11 waiting for isolate _isolateMain to check in
Attempt:12 waiting for isolate vm-isolate to check in
Attempt:12 waiting for isolate _isolateMain to check in
Attempt:13 waiting for isolate vm-isolate to check in
Attempt:13 waiting for isolate _isolateMain to check in
Attempt:14 waiting for isolate vm-isolate to check in
Attempt:14 waiting for isolate _isolateMain to check in
...
```